src/agentLibrary.core.js

```diff
@@ -227,12 +230,23 @@ function initAgentLibraryCore (context) {
             this.callbacks = config.callbacks;
         }
 
-        if(typeof config.socketDest !== 'undefined'){
+        if(typeof config.authHost !== 'undefined'){
+            UIModel.getInstance().authHost = config.authHost;
+        }
+
+        if(typeof config.localTesting !== 'undefined'){
```

Comment:
change this to truthy check
```suggestion
        if(!!config.localTesting){
```

Reply:
done